src/test/mocks/archiveLoader.ts

```diff
@@ -76,6 +77,7 @@ export default () => {
       localBooks[`${parentBook.id}@${parentBook.version}`].tree.contents.push({
         id: `${newPage.id}@${newPage.version}`,
         shortId: `${newPage.shortId}@${newPage.version}`,
+        slug: getUrlParamForPageId(parentBook, newPage.title),
```

Comment:
since this is generating the tree for this mock book, i don't think its good to rely on calculating the new slug. maybe we should require passing the slug as a third param?